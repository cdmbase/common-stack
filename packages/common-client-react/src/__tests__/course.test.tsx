import * as React from 'react';
import { shallow } from 'enzyme';
import Course from '../connector/course';

describe('Test on Course component', () => {
    const props = {
        history: jest.fn()
    };
    it('should render Link component', () => {
        const wrapper = shallow(<Course {...props} />);
        expect(wrapper).toMatchSnapshot();
    });
});