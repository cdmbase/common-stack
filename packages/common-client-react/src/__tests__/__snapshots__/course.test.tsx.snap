// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test on Course component should render Link component 1`] = `
<div>
  <Link
    0="l"
    1="o"
    10="3"
    11="0"
    12="0"
    13="0"
    14="/"
    2="c"
    3="a"
    4="l"
    5="h"
    6="o"
    7="s"
    8="t"
    9=":"
    replace={false}
    to="undefined"
  >
    Page: undefined
  </Link>
</div>
`;
